### DISCUSSION TITLE
Capella -SLC file phase extraction

### DISCUSSION THREAD

**Discussion by koles289** [sertit/eoreader#77] [Q&A] [answered]
Hi everyone,

I am a new-baby in Satellite imagery processing and I have one question. I need to extract phase information from Capella  SLC file. Does **eoreader** supports  any functions that would help me to extract it?
I was trying to load the data using the _load_ function but that one is working only for GEO pictures in capella dataset, not for SLC.
I would be grateful for any tricks and tips :)

**Comment by remi-braun** [comment, marked as answer]
Hello,

Sadly, EOReader is focused only on intensity processes, and is not fit for retrieving the phase.
If you give SLC data to EOReader, it will automatically convert it to ground range.

**Reply by remi-braun** [reply]
However, you can use directly ESA SNAP software, or some wrapper in Python such as snapista :)

**Reply by koles289** [reply]
Ok, thank you for an answer :)
